docs/RELEASE_NOTES.md

```diff
@@ -38,6 +38,44 @@
 
 -->
 
+### 7.0.0 - Aug 23, 2021
+[Commits](https://github.com/commercetools/commercetools-sync-java/compare/6.0.0...7.0.0) |
+[Javadoc](https://commercetools.github.io/commercetools-sync-java/v/7.0.0/) |
+[Jar](https://search.maven.org/artifact/com.commercetools/commercetools-sync-java/7.0.0/jar)
+
+- 🚧 **Breaking Changes** (1)
+  - **Dependency management:** Avoid JVM-SDK libraries version mismatch between projects.
+     You should not explicitly add JVM-SDK dependencies into your project where you are using Sync-Java. Please remove them if you have already added the below dependencies in your project.
```

Comment:
What happens if users do set explicitly the deps? Also should this note be explicitly written in the main README?

Reply:
I see this lib as prerequisite here: https://github.com/commercetools/commercetools-sync-java#prerequisites

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/761/commits/852e9030cdc0b1fbfde9becd9c033140ab178be7